src/sentry/utils/suspect_resolutions_releases/get_suspect_resolutions_releases.py

```diff
@@ -0,0 +1,51 @@
+from typing import Sequence
+
+from sentry import features
+from sentry.models import Group, GroupRelease, GroupStatus, Project, Release, ReleaseProject
+from sentry.signals import release_created
+from sentry.tasks.base import instrumented_task
+from sentry.utils.suspect_resolutions_releases import ALGO_VERSION, analytics
+
+
+@release_created.connect(weak=False)
+def record_suspect_resolutions_releases(release, project, **kwargs) -> None:
+    if features.has("projects:suspect-resolutions-releases", project):
+        get_suspect_resolutions_releases.delay(release, project)
+
+
+@instrumented_task(
+    name="sentry.tasks.get_suspect_resolutions_releases", queue="get_suspect_resolutions_releases"
+)
+def get_suspect_resolutions_releases(release: Release, project: Project) -> Sequence[int]:
+    suspect_resolution_issue_ids = []
+    is_suspect_resolution = False
+
+    releases_in_project = ReleaseProject.objects.filter(project=project).values_list("release_id")
+    latest_release = Release.objects.filter(id__in=releases_in_project).order_by("-date_added")[1]
```

Comment:
Since multiple projects can be associated with a release, I think you'll have to rework this query to support that.

Maybe something like:
```
if release.projects:
  prev_release_projects = ReleaseProject.objects.filter(project_id__in=[p.id for p in release.projects], release__date_added__lt=self.date_added)
              .order_by("-release__date_added")
  releases_by_project:Mapping[int, Sequence[Release]] = defaultdict(list)
  for rp in prev_release_projects:
    if prev_release_projects.get(rp.project.id):
      prev_release_projects.get(rp.project.id).append(rp.release)

  latest_release_per_project = {
            user_id: max(r_list, key=lambda r: r.date_added)
            for project_id, r_list in prev_release_projects.items()
        }
```